Food_Delivery.sql

```diff
@@ -0,0 +1,120 @@
+CREATE TABLE Restaurants(
+	RestaurantId INT PRIMARY KEY,
+	Name VARCHAR(30),
+	Email VARCHAR(50),
+	PhoneNumber VARCHAR(12),
+	City VARCHAR(30),
+	Street VARCHAR(30),
+	Country VARCHAR(30)
+);
+
```

Comment:
Additional table considerations 

 table for restaurant categories/cuisines
 table for menu item availability scheduling
 tracking of order modifications/cancellations
 driver earnings/commission tracking
 table for customer favorite items/restaurants